{"version":3,"sources":["../../../app/v1/middlewares/errors.js"],"names":["error","controller","req","res","next","status","json","message"],"mappings":";;;;;;;;;;;;;AAAA,IAAMA,KAAK,GAAG,SAARA,KAAQ,CAACC,UAAD;AAAA;AAAA,6FAAgB,iBAAOC,GAAP,EAAYC,GAAZ,EAAiBC,IAAjB;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA,qBAEbH,UAAU,CAACC,GAAD,EAAMC,GAAN,EAAWC,IAAX,CAFG;;AAAA;AAAA;;AAAA;AAAA;AAAA;AAAA,+CAInBD,GAAG,CAACE,MAAJ,CAAW,GAAX,EAAgBC,IAAhB,CAAqB;AAC1BC,gBAAAA,OAAO,EAAE;AADiB,eAArB,CAJmB;;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA,KAAhB;;AAAA;AAAA;AAAA;AAAA;AAAA,CAAd;;eAUeP,K","sourcesContent":["const error = (controller) => async (req, res, next) => {\n  try {\n    return await controller(req, res, next);\n  } catch (err) {\n    return res.status(500).json({\n      message: \"Ha ocurrido un error al intentar acceder a la base de datos\",\n    });\n  }\n};\n\nexport default error;\n"],"file":"errors.js"}